components/kern-table/CellComponents.tsx

```diff
@@ -243,4 +244,44 @@ function LabelCell({ sourceContainer }) {
     </span>
 }
 
-export { OrganizationAndUsersCell, MaxRowsColsCharsCell, CommentsCell, ExportConsumptionAndDeleteCell, BadgeCell, OrganizationUserCell, DeleteUserCell, LevelCell, ArchiveReasonCell, ProjectNameTaskCell, CancelTaskCell, IconCell, ConfigCell, EditDeleteOrgButtonCell, ViewStackCell, AbortSessionButtonCell, FeedbackMessageCell, FeedbackMessageTextCell, JumpToConversationCell, RemoteVersionCell, ExternalLinkCell, ModelDateCell, FileSizeCell, StatusModelCell, DeleteModelCell, LabelCell }
\ No newline at end of file
+function ViewCell({ onClick, disabled }) {
+    const clickView = useCallback(() => {
+        if (onClick) onClick();
+    }, [onClick]);
+    return <KernButton text="View" onClick={clickView} disabled={disabled} />
+}
+
+function EvaluationRunStateCell({ value }) {
+    const color = useMemo(() => {
```

Comment:
maybe we should memo the whole color class so it doesnt need to be calculated later